data/json/items/tool/explosives.json

```
@@ -641,7 +641,7 @@
     "type": "TOOL",
     "category": "weapons",
     "name": "black gunpowder charge",
-    "description": "This is a home-made explosive device, consisting of a large plastic jug filled with black gunpowder and scrap metal, equipped with a long fuse.  Use this item to light the fuse.  Should explode in a few minutes�",
+    "description": "This is a home-made explosive device, consisting of a large plastic jug filled with black gunpowder and scrap metal, equipped with a long fuse.  Use this item to light the fuse.  Should explode in a few minutes...",
```

Using the Unicode ellipsis instead.
```suggestion
    "description": "This is a home-made explosive device, consisting of a large plastic jug filled with black gunpowder and scrap metal, equipped with a long fuse.  Use this item to light the fuse.  Should explode in a few minutes…",
```